.github/workflows/end2end.yaml

```
@@ -120,6 +120,33 @@ jobs:
       - name: Verify monitoring dashboard versions
         run: bash ./.github/scripts/check_versions.sh
 
+  check-workflows:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+          cache: yarn
+          cache-dependency-path: tests/workflows/yarn.lock
+      - name: Install dependencies
+        run: yarn --cwd tests/workflows install
+      - name: Run workflows tests
+        run: ACT_LOG=true yarn --cwd tests/workflows test
+      - name: Collect logs
+        run: mkdir -p act/ && mv tests/workflows/act-*.log act/
```

```suggestion
        run: |
            mkdir -p /tmp/artifacts/data/check-workflows/test-logs/ && mv tests/workflows/act-*.log mkdir -p /tmp/artifacts/data/check-workflows/test-logs/
            tar zcvf /tmp/artifacts/${{ github.sha }}-check-workflows-logs-volumes.tgz /tmp/artifacts/data/check-workflows/test-logs;
```

Should we keep the same artifacts structure we use for logs of the other stages ?